tests/test_ping_json.py

```diff
@@ -0,0 +1,36 @@
+import requests
+
+
+def json_health_check():
```

Comment:
test function should start with prefix 'test_' so pytest knows to run it as a test